src/test/test_ipaddr.cc

```diff
@@ -341,18 +339,13 @@ TEST(CommonIPAddr, TestV6_SkipLoopback)
   result =
     find_ip_in_subnet_list(nullptr, (struct ifaddrs*)&one,
                            CEPH_PICK_ADDRESS_IPV4 | CEPH_PICK_ADDRESS_IPV6,
-                           "2001:1234:5678:90ab::0/64", "", true);
+                           "2001:1234:5678:90ab::0/64", "");
```

Comment:
Same here. We need these tests to confirm that non-loopback are not chose even when we don't specify a subnet.

Reply:
@dvanders thank you for your review. fixed and repushed. could you please take another look?